### ISSUE TITLE
Add ability to hide SwaggerUI v3 header

### ISSUE THREAD

**Issue by BlackGad** [RicoSuter/NSwag#2777]
It would be nice to have ability to hide Top panel with explore button.

[Accordingly to existing swagger issue](https://github.com/swagger-api/swagger-ui/issues/3229) several ways to do this:

- [First](https://github.com/swagger-api/swagger-ui/issues/3229#issuecomment-425739134)
- [Second](https://github.com/swagger-api/swagger-ui/issues/3229#issuecomment-396442225)

Files where UI configuration must be mapped 
- [Owin](https://github.com/RicoSuter/NSwag/tree/master/src/NSwag.AspNet.Owin/SwaggerUi3/index.html)
- [AspNetCore](https://github.com/RicoSuter/NSwag/tree/master/src/NSwag.AspNetCore/SwaggerUi3/index.html)

**Comment by RicoSuter** [comment]
So it’s just a swagger ui option which is not exposed by nswag?

**Comment by BlackGad** [comment]
Yeah, you have already added additional options but this case needs some additional manipulations in javascript options instance.
